tools/check-typo

```diff
@@ -35,27 +35,9 @@
 # which lists the rules that should be disabled for this file.
 # The rule names are the ones shown above in parentheses.
 
-# Built-in exceptions:
+# Built-in exception:
 # - Any file git identifies as binary
 #   is automatically exempt from all the rules.
-# - Any file whose name matches one of the following patterns is
-#   automatically exempt from all rules
-#     *.reference
-#     *.opt_reference
```

Comment:
Just FTR, `*.opt_reference` was removed with the ocamltest migration (gone since 8c2d90543fce1e5aca1bd9bc796a51f04be48b4f)